docs/docusaurus.config.js

```diff
@@ -36,6 +36,9 @@ module.exports = {
     footer: {
       copyright: `Copyright © ${new Date().getFullYear()} The OpenEBS Authors | All rights reserved`,
     },
+    prism: {
+      theme: Object.assign(require('prism-react-renderer/themes/github'), { plain: { color: '#ffffff', backgroundColor: '#3B4473' }})
```

Comment:
Is there a possibility of separating the style object by means of a reference ?

Reply:
we can create the object on the top section of `docusaurus.config.js` and merge that object at line 40.